app/models/rbac.rb

```diff
@@ -239,8 +240,8 @@ def self.get_managed_filter_object_ids(klass, scope, filter)
   end
 
   def self.find_targets_with_direct_rbac(klass, scope, rbac_filters, find_options = {}, user_or_group = nil)
-    filtered_ids, u_filtered_ids = calc_filtered_ids(klass, scope, rbac_filters, user_or_group)
-    find_targets_filtered_by_ids(klass, scope, find_options, u_filtered_ids, filtered_ids)
+    filtered_ids, u_filtered_ids = calc_filtered_ids(scope, rbac_filters, user_or_group)
```

Comment:
I'm guessing you mean `klass` in `find_targets_with_direct_rbac`.

Done.

The long term goal is to get rid of `klass` from everywhere